Description: Prevent the removal of udev rules by agent
 Agent violates packaging rules by modifying the files from another
 package. This patch makes the package compliant.
Author: Ben Howard
Last-Update: 2013-07-11
--- a/waagent
+++ b/waagent
@@ -87,8 +87,7 @@
 VMM_STARTUP_SCRIPT_NAME='install'
 VMM_CONFIG_FILE_NAME='linuxosconfiguration.xml'
 global RulesFiles
-RulesFiles = [ "/lib/udev/rules.d/75-persistent-net-generator.rules",
-               "/etc/udev/rules.d/70-persistent-net.rules" ]
+RulesFiles = []
 VarLibDhcpDirectories = ["/var/lib/dhclient", "/var/lib/dhcpcd", "/var/lib/dhcp"]
 EtcDhcpClientConfFiles = ["/etc/dhcp/dhclient.conf", "/etc/dhcp3/dhclient.conf"]
 global LibDir
